### PR TITLE
Translate existing hardcoded english

### PR DESCRIPTION
**Pull request by siegfault**
I'm happy to change any keys here, but am least confident about the compound keys (ATTRIBUTES_AND_STATUS_EFFECTS & EXPERIENCE_AND_LEVEL). Any thoughts there are much appreciated!

### CODE REVIEW

File: src/actor/character/sheets-tabs/main-tab.hbs [hunk 1]
```diff
@@ -1,92 +1,99 @@
...
       </div>
     </div>
 
     <div class="traits-section">
-      <h3>Traits <i class="fas fa-fingerprint"></i> </h3>
+      <h3>{{localize "BIO.TRAITS"}}<i class="fas fa-fingerprint"></i> </h3>
       <div class="traits">
         <div class="trait-with-answer">
-          <h4>Identity:
+          <h4>{{localize "BIO.IDENTITY"}}:
```
Comment (mbenfowler) [inline src/actor/character/sheets-tabs/main-tab.hbs:88]: All punctuation, including colons should be within the localized string!

File: src/actor/character/sheets-tabs/main-tab.hbs [hunk 2]
```diff
@@ -1,92 +1,99 @@
...
             <input name="system.bio.identity.value" type="text" value="{{this.actor.bio.identity.value}}" />
           </h4>
         </div>
         <div class="trait-with-answer">
-          <h4>Theme:
+          <h4>{{localize "BIO.THEME"}}:
```
Comment (mbenfowler) [inline src/actor/character/sheets-tabs/main-tab.hbs:93]: colon

File: src/actor/character/sheets-tabs/main-tab.hbs [hunk 3]
```diff
@@ -1,92 +1,99 @@
...
             <input name="system.bio.theme.value" type="text" value="{{this.actor.bio.theme.value}}" />
           </h4>
-          <h4>Origin:
+          <h4>{{localize "BIO.ORIGIN"}}:
```
Comment (mbenfowler) [inline src/actor/character/sheets-tabs/main-tab.hbs:96]: colon

File: src/actor/character/sheets-tabs/main-tab.hbs [hunk 4]
```diff
@@ -1,92 +1,99 @@
...
+      <h4>
+        {{localize "BIO.HIT_POINTS"}}:
+        <input id="number-input" type="number" name="system.baseAttribute.baseHitPoints.value" value="{{this.actor.attributes.baseHitPoints.value}}" /> / <input id="number-input" type="number" name="system.baseAttribute.baseHitPoints.max" value="{{this.actor.attributes.baseHitPoints.max}}" />
+      </h4>
     </div>
 
     <div class="points mind-points">
-      <h4> Mind Points: <input id="number-input" type="number" name="system.baseAttribute.baseMindPoints.value" value="{{this.actor.attributes.baseMindPoints.value}}" /> / <input id="number-input" type="number" name="system.baseAttribute.baseMindPoints.max" value="{{this.actor.attributes.baseMindPoints.max}}" /> </h4>
+      <h4>
+        {{localize "BIO.MIND_POINTS"}}:
```
Comment (mbenfowler) [inline src/actor/character/sheets-tabs/main-tab.hbs:19]: colon

File: src/actor/character/sheets-tabs/main-tab.hbs [hunk 5]
```diff
@@ -1,92 +1,99 @@
...
+      <h3>
+        {{localize "BIO.PRONOUNS"}}:
         <input id="dark-text" style="width: 100px" name="system.bio.pronouns.value" type="text" value="{{this.actor.bio.pronouns.value}}" />
       </h3>
     </div>
 
     <div class="points hit-points">
-      <h4> Hit Points: <input id="number-input" type="number" name="system.baseAttribute.baseHitPoints.value" value="{{this.actor.attributes.baseHitPoints.value}}" /> / <input id="number-input" type="number" name="system.baseAttribute.baseHitPoints.max" value="{{this.actor.attributes.baseHitPoints.max}}" /> </h4>
+      <h4>
+        {{localize "BIO.HIT_POINTS"}}:
```
Comment (mbenfowler) [inline src/actor/character/sheets-tabs/main-tab.hbs:12]: colon

File: src/actor/character/sheets-tabs/main-tab.hbs [hunk 6]
```diff
@@ -1,92 +1,99 @@
 <div class="main-tab">
   <div class="left-content">
     <div class="pronouns">
-      <h3> Pronouns:
+      <h3>
+        {{localize "BIO.PRONOUNS"}}:
```
Comment (mbenfowler) [inline src/actor/character/sheets-tabs/main-tab.hbs:5]: colon

File: src/actor/character/sheets-tabs/main-tab.hbs [hunk 7]
```diff
@@ -1,92 +1,99 @@
...
-      <h4> Mind Points: <input id="number-input" type="number" name="system.baseAttribute.baseMindPoints.value" value="{{this.actor.attributes.baseMindPoints.value}}" /> / <input id="number-input" type="number" name="system.baseAttribute.baseMindPoints.max" value="{{this.actor.attributes.baseMindPoints.max}}" /> </h4>
+      <h4>
+        {{localize "BIO.MIND_POINTS"}}:
+        <input id="number-input" type="number" name="system.baseAttribute.baseMindPoints.value" value="{{this.actor.attributes.baseMindPoints.value}}" /> / <input id="number-input" type="number" name="system.baseAttribute.baseMindPoints.max" value="{{this.actor.attributes.baseMindPoints.max}}" />
+      </h4>
     </div>
 
     <div class="attributes-and-status-effects">
-      <h3 class="header"> Attributes and Status Effects <i class="fas fa-hand-fist"></i></h3>
+      <h3 class="header">{{localize "BIO.ATTRIBUTES_AND_STATUS_EFFECTS"}}<i class="fas fa-hand-fist"></i></h3>
```
Comment (mbenfowler) [inline src/actor/character/sheets-tabs/main-tab.hbs:25]: I think these compound names are just fine!

**Review by mbenfowler — CHANGES_REQUESTED** [review]
colonblow

File: src/actor/character/sheets-tabs/main-tab.hbs (hunk 1 above)
Comment (siegfault) [reply src/actor/character/sheets-tabs/main-tab.hbs:88]: TIL: https://softwareengineering.stackexchange.com/a/234554. Thanks! I'll make that change shortly

**Review by mbenfowler — APPROVED** [review]

**Pull request merged**